src/types/profile.js

```diff
@@ -211,6 +213,20 @@ export type PausedRange = {
   reason: 'profiler-paused' | 'collecting',
 };
 
+export type JsTracerTable = {
+  events: JsTracerEvents,
+  stringTable: UniqueStringArray,
+};
+
+export type JsTracerEvents = {
+  events: Array<IndexIntoJsTracerStringTable>,
+  timestamps: Array<Microseconds>,
+  durations: Array<Microseconds | -1>,
+  lines: Array<number | -1>, // Line number.
+  columns: Array<number | -1>, // Column number.
```

Comment:
Yes, that's true, I think we did this while working on the export data. I'm having trouble recalling our exact reasoning, but I think it had to do with file sizes. I could convert them in the profile processing. @dpalmeiro do you have opinions here? Is there benefits to keeping this -1 in the processed profile? The can get uploaded to our profile service.

Reply:
Yes, it was for file size but I'll change it to null in bug 1497016.  Same goes for events with no duration,  I'll change that from -1 to null.

Reply:
I will make this change.

Reply:
I made this change, but note that in the `getJsTracerLeafTiming` algorithm, I left the local variable `prefixesTip` as a -1, because I felt it made the code simpler. Flow can't refine mutable variables very well, so -1 was more reasonable as the code exists. I haven't read the rest of the review of the code yet thought, so we'll see if it sticks :)